zipline/algorithm.py

```diff
@@ -1461,8 +1888,13 @@ def validate_account_controls(self):
 
     @api_method
     def set_max_leverage(self, max_leverage=None):
-        """
-        Set a limit on the maximum leverage of the algorithm.
+        """Set a limit on the maximum leverage of the algorithm.
+
+        Parameters
+        ----------
+        max_leverage : float, optional
+            The maximum leverage for the algorithm. If not provided there will
+            be no maximum.
```

Comment:
In fact, looks like a [ValueError will be raised] (https://github.com/quantopian/zipline/blob/16fd6681a66101d3f06a0bb639ee699a8905ceda/zipline/finance/controls.py#L381-L384) later.